docs/sources/features/datasources/stackdriver.md

```diff
@@ -86,58 +86,62 @@ If Grafana is running on a Google Compute Engine (GCE) virtual machine, it is po
 
 Read more about creating and enabling service accounts for GCE VM instances [here](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances).
 
-## Metric Query Editor
+## Using the Query Editor
 
-{{< docs-imagebox img="/img/docs/v67/stackriver-query-editor.png" max-width= "400px" class="docs-image--right" >}}
+The Stackdriver query editor allows you to build two types of queries - **Metric** and **Service Level Objective (SLO)**. Both types return time series data.
 
-The Stackdriver query editor allows you to select metrics, group/aggregate by labels and by time, and use filters to specify which time series you want in the results.
+### Metric Queries
+
+{{< docs-imagebox img="/img/docs/v70/metric-query-builder.png" max-width= "400px" class="docs-image--right" >}}
+
+The metric query editor allows you to select metrics, group/aggregate by labels and by time, and use filters to specify which time series you want in the results.
 
 Begin by choosing a `Project`. Then select a `Service` and then a metric from the `Metric` dropdown. Use the plus and minus icons in the filter and group by sections to add/remove filters or group by clauses.
```

Comment:
Before one selecting `Project` needs to select `QueryType`.

Reply:
👍